doc/source/whatsnew/v0.20.0.txt

```diff
@@ -271,6 +271,7 @@ Performance Improvements
 - Improved performance of ``pd.wide_to_long()`` (:issue:`14779`)
 - Increased performance of ``pd.factorize()`` by releasing the GIL with ``object`` dtype when inferred as strings (:issue:`14859`)
 
+- When reading buffer object in ``read_sas()`` method without specified format, filepath string is inferred rather than buffer object. Error is now thrown if buffer object is provided without format {sas7bdat|xport} specification.
```

Comment:
just the first sentence.

put in bug fix section; use this PR number as the issue number.